docs/docs/references/configuration/cli/trivy_sbom.md

```diff
@@ -42,6 +42,7 @@ trivy sbom [flags] SBOM_PATH
       --no-progress                 suppress progress bar
       --offline-scan                do not issue API requests to identify dependencies
   -o, --output string               output file name
+      --parallel int                number of goroutines enabled for parallel scanning (default 5)
```

Comment:
Looks like we don't use parallel for SBOM target.
Perhaps we can disable this flag.

Reply:
Nice catch. Done https://github.com/aquasecurity/trivy/pull/5572/commits/ca71aace803fd20f6115bdab58e762a4938a0b91